lib/gps.js

```diff
@@ -0,0 +1,439 @@
+//var IS_TEST_MODE = !!process.env.IS_TEST_MODE;
+var Board = require("./board");
+var events = require("events");
+var util = require("util");
+var Pin = require("./pin");
+
+var priv = new Map();
+
+var Breakouts = {
+
+  /*
+   * https://www.adafruit.com/products/746
+   */
+  ADAFRUIT_ULTIMATE_GPS: {
+    receiver: {
+      value: "FGPMMOPA6H"
+    }
+  }
+
+};
+
+// GPS Antenna Modules
+var Receivers = {
+
+  /*
+   * http://www.gtop-tech.com/en/product/LadyBird-1-PA6H/MT3339_GPS_Module_04.html
+   */
+  FGPMMOPA6H: {
+    // Later, when we add logging that code will go here
+    chip: {
+      value: "MT3339"
+    }
+  }
+
+};
+
+// GPS chips
+var Chips = {
+
+  DEFAULT: {
+    baud: {
+      value: 9600
+    },
+    configure: {
+      value: function(callback) {
+        callback();
+      }
+    }
+  },
+
+  /*
+   * http://www.mediatek.com/en/products/connectivity/gps/mt3339/
+   */
+  MT3339: {
+    baud: {
+      value: 9600
+    },
+    configure: {
+      value: function(callback) {
+        callback();
+      }
+    },
```

Comment:
I forgot to mention this before—I really like what you're doing here, really excellent foresight. One thing I'm thinking of now, should this be async?

```js   
process.nextTick(callback);
```

It seems like future configuration operations may very well be the kind that require some degree of async behavior—we might as well enforce async now, even if nothing is actually happening